### PR TITLE
Blog post about concurrent uni in Mutiny

### PR DESCRIPTION
**Pull request by cescoffier**

### CODE REVIEW

File: _posts/2020-09-29-mutiny-concurrent-uni.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+layout: post
+title: 'Concurrent asynchronous actions with Mutiny'
+date: 2020-09-29
+tags: reactive mutiny async
+synopsis: Running concurrent asynchronous actions with Mutiny
+author: cescoffier
+---
+
+This week, I've been asked about a widespread use case around concurrency.
+This user wanted to call two microservices _in parrallel_, and when both results are received, join them and continue the processing.
```
Comment (kenfinnigan) [inline _posts/2020-09-29-mutiny-concurrent-uni.adoc:11]: s/parrallel/parallel

File: _posts/2020-09-29-mutiny-concurrent-uni.adoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+No matter which client we use, we need some remote services to call.
+Let's use:
+
+* https://programming-quotes-api.herokuapp.com/ - to get a quote about programming
+* https://api.chucknorris.io/ - to get a quote about _Chuck Norris_
+
+
+First thing first, let's see the code required to retrieve our quotes:
+While both services a similar, the structure of the response differs a little bit.
```
Comment (kenfinnigan) [inline _posts/2020-09-29-mutiny-concurrent-uni.adoc:36]: s/services a/services are

File: _posts/2020-09-29-mutiny-concurrent-uni.adoc (hunk 1 above)
Comment (jponge) [inline _posts/2020-09-29-mutiny-concurrent-uni.adoc:11]: parallel

File: _posts/2020-09-29-mutiny-concurrent-uni.adoc [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+[source, java]
+----
+Uni<Tuple2<String, String>> tuple = Uni.combine().all()
+    .unis(getProgrammingQuote(client), getChuckNorrisQuote(client))
+    .asTuple();
+----
+
+When someone subscribes to the `Uni tuple`, it subscribes to the `getProgrammingQuote(client)` and `getChuckNorrisQuote(client)` `Unis`, which invoke the services.
+So the requests are emitted, and the services and invoked concurrently.
```
Comment (kenfinnigan) [inline _posts/2020-09-29-mutiny-concurrent-uni.adoc:78]: s/and invoked/are invoked

File: _posts/2020-09-29-mutiny-concurrent-uni.adoc [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    .asTuple();
+----
+
+When someone subscribes to the `Uni tuple`, it subscribes to the `getProgrammingQuote(client)` and `getChuckNorrisQuote(client)` `Unis`, which invoke the services.
+So the requests are emitted, and the services and invoked concurrently. 
+
+When both responses are available, it combines them into a _Tuple_, a simple structure carrying multiple items.
+
+In other words, concurrently calling our services is pretty straightforward.
+Just creates the `Unis` representing the services or the asynchronous action you want to achieve and combine them using `Uni.combine().all()`
```
Comment (kenfinnigan) [inline _posts/2020-09-29-mutiny-concurrent-uni.adoc:83]: s/creates/create

**Review by jponge — CHANGES_REQUESTED** [review]
Typo :-)

**Review by kenfinnigan — CHANGES_REQUESTED** [review]

**Comment by cescoffier** [comment]
@kenfinnigan anything else to add?

**Review by kenfinnigan — APPROVED** [review]
sorry!

**Pull request merged**